Dockerfile.alpine

```diff
@@ -55,6 +55,7 @@ RUN \
     && mkdir -p "/var/lib/snipeit/dumps" && rm -r "/var/www/html/storage/app/backups" && ln -fs "/var/lib/snipeit/dumps" "/var/www/html/storage/app/backups" \
     && mkdir -p "/var/lib/snipeit/keys" && ln -fs "/var/lib/snipeit/keys/oauth-private.key" "/var/www/html/storage/oauth-private.key" \
     && ln -fs "/var/lib/snipeit/keys/oauth-public.key" "/var/www/html/storage/oauth-public.key" \
+    && chown -h "/var/www/html/storage/*.key"
```

Comment:
There is a missing ```\``` at the end of the line and chown is missing the user who should own the file.

When I tried changing it to ```    && chown -h apache "/var/www/html/storage/*.key"``` this fails with ```chown: /var/www/html/storage/*.key: No such file or directory```

Any ideas? Thanks!

Reply:
Total miss here in the initial PR. This should match `Dockerfile`. Pretty sure you want the new owner to be the `docker` user.

Reply:
@smcpeck I can try that tomorrow, I missed the title of the PR whoops.    I was running into the error that chown was failing because the files didn't exist but will report back when trying to change to the correct user.

Reply:
@smcpeck the user docker doesn't exist in Dockerfile.alpine